package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "broccoli-lint-remark",
+  "version": "1.0.0",
+  "description": "broccoli wrapper for remark-lint",
+  "main": "lib/index.js",
+  "scripts": {
+    "lint": "eslint .",
+    "test": "mocha test/*test.js test/**/*test.js"
```

Comment:
And why not with option ` --compilers js:babel-core/register -r babel-polyfill` and so you can use new imports etc.? Of course, You should add babel also